build.gradle

```diff
@@ -14,7 +14,7 @@ buildscript {
         lifecycleVersion = '2.5.1'
         androidXArchTestVersion = '2.0.0'
         paletteKtxVersion = '1.0.0'
-        jsonhandleviewVersion = '1.2.2'
+        jsonhandleviewVersion = '1.2.2-jetified'
```

Comment:
This should have been `1.2.3-SNAPSHOT`...but anyway, you can upload a new `1.2.3` release version and update this

Reply:
updated